### ISSUE TITLE
Add construct_subsegment_by_recordingchannel to Segment

### ISSUE THREAD

**Issue by mdenker** [INM-6/python-neo#8]
A function like this would be useful:

```
h.construct_subsegment_by_recordingchannel(rc_list=[k])
```

**Comment by lphan** [comment]
@mdenker Can you describe a bit more what does this function ? and where to find it or even a short example how to start it, how is 'h' initiated ?

**Comment by alperyeg** [comment]
I think this is a feature Michael wants to have in neo, that's why the tag is enhancement. 
I believe this function should have a similar functionality like `construct_subsegment_by_unit`. Probably it should be in `Segment`, Michael needs to clarify that.

**Comment by mdenker** [comment]
Yes, that's what I was thinking. In the mean time, this must be reevaluated in the context of ChannelIndexes...
